lib/Interpreter/Functions/Nextcloud/Get_Tags.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace OCA\FilesScripts\Interpreter\Functions\Nextcloud;
+
+use OC\SystemTag\SystemTagManager;
+use OC\SystemTag\SystemTagObjectMapper;
+use OCA\FilesScripts\Interpreter\RegistrableFunction;
+use OCP\Files\InvalidPathException;
+use OCP\Files\NotFoundException;
+
+/**
+ * `get_tags(Node file): array`
+ *
+ * Gets tags of a file. Returns list of tags.
+ *
+ * ```lua
+ * local tags = get_tags(get_input_files()[1])
+ *
+ * ```
+ */
+class Get_Tags extends RegistrableFunction {
```

Comment:
I would suggest we include "file" in the name somewhere.

Some suggestions: `Get_Tags_From_File` or `Get_File_Tags`